File: old.eFromAPI/Plugins/Customers.Pn/Customers.Pn/Infrastructure/Models/Customer/CustomersPnRequestModel.cs
namespace Customers.Pn.Infrastructure.Models.Customer
{
    public class CustomersPnRequestModel
    {
        public string SortColumnName { get; set; }
        public int Offset { get; set; }
        public int PageSize { get; set; }
        public bool IsSortDsc { get; set; }
    }
}

File: eform-client/src/app/plugins/modules/customers-pn/models/customer/customers-pn-request.model.ts
export class CustomersPnRequestModel {
  name: string;
  sortColumnName: string;
  pageIndex: number;
  offset: number;
  pageSize: number;
  isSortDsc: boolean;

  constructor() {
    this.sortColumnName = 'Id';
    this.isSortDsc = true;
    this.pageSize = 5;
    this.pageIndex = 1;
    this.offset = 0;
  }
}
